### PR TITLE
Release 0.11

### PR DESCRIPTION
**Pull request by mattwthompson**
I need this so I'm just trying to snipe the bot, and possibly fix issues myself

Not seeing huge changes in the diff between versions: https://github.com/computationalmodelling/nbval/compare/0.10.0...0.11.0

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mattwthompson** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -18,11 +18,11 @@ build:
...
 requirements:
   host:
-    - python >=3.6
+    - python >=3.7
     - pip
   run:
-    - python >=3.6
-    - pytest
+    - python >=3.7
+    - pytest >=7
```
Comment (mattwthompson) [inline recipe/meta.yaml:25]: https://github.com/computationalmodelling/nbval/blob/0.11.0/setup.py#L23-L30

**Comment by takluyver** [comment]
Thanks for dealing with the conda-forge part!

**Comment by mattwthompson** [comment]
Thank you!

**Pull request merged**